packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -409,7 +409,11 @@ const conf = (module.exports = convict({
       },
       enabled_client_ids: {
         // 123done enabled for functional tests, 321done is not.
-        default: ['dcdb5ae7add825d2', '7f368c6886429f19'],
+        default: [
+          'dcdb5ae7add825d2',
+          '7f368c6886429f19',
+          '5882386c6d801776', // Firefox Desktop
```

Comment:
Should this be done as a cloudops deployment?

Reply:
they don't use that ENV var yet (I couldn't find it)

cc @jrgm